packages/sdk/src/evm/types/any-evm/deploy-data.ts

```diff
@@ -29,9 +28,24 @@ export type DeployedContractType =
   | "infra"
   | "implementation"
   | "create2Factory"
+  | "plugin"
   | "custom";
 
 export type DeploymentTransaction = {
-  contractType: DeployedContractType;
+  contractType: DeployedContractType | string;
```

Comment:
why do you need `| string` here? can we not just add to `DeployedContractType` ?

Reply:
String is for `preset`.
Since it is a broad grouping for implementation, infra, etc, didn't add it to the contract types.